examples/network_state_example.yml

```diff
@@ -19,16 +19,6 @@
       ansible.builtin.include_role:
         name: linux-system-roles.network
 
-    - name: Delete the ethernet device eth1
-      vars:
-        network_state:
-          interfaces:
-            - name: eth1
-              type: ethernet
-              state: absent
-      ansible.builtin.include_role:
-        name: linux-system-roles.network
-
```

Comment:
@tyll , it is worth explaining...

For state apply, nmstate unifies the veth and ethernet into `type: ethernet`, the veth type is only meaningful for querying the state.  We can only use nmstate to delete the virtual NIC that is created by NM/Nmstate, we can not use nmstate to delete the veth device that is created by iproute2.